src/autopas/containers/CellBlock3D.h

```diff
@@ -106,127 +106,121 @@ class CellBlock3D : public CellBorderAndFlagManager {
   void reserve(size_t numParticles);
 
   /**
-   * rebuild the cellblock. This resizes the cellblock and sets the appropriate
-   * internal variables
-   * @param vec new vector of ParticleCells to which the internal pointer is set
-   * @param bMin new lower corner of the cellblock
-   * @param bMax new higher corner of the cellblock
-   * @param interactionLength max. radius of interaction between particles
-   * @param cellSizeFactor cell size factor relative to interactionLength
+   * Rebuild the cell block. This resizes the cell block and sets the appropriate internal variables.
+   * @param vec new vector of ParticleCells to which the internal pointer is set.
+   * @param bMin new lower corner of the cell block.
+   * @param bMax new higher corner of the cell block.
+   * @param interactionLength max. radius of interaction between particles.
+   * @param cellSizeFactor cell size factor relative to interactionLength.
```

Comment:
```suggestion
   * @param vec New vector of ParticleCells to which the internal pointer is set.
   * @param bMin New lower corner of the cell block.
   * @param bMax New higher corner of the cell block.
   * @param interactionLength Max. radius of interaction between particles.
   * @param cellSizeFactor Cell size factor relative to interactionLength.
```

Since you already changed this everywhere else :D